addons/project_timesheet_holidays/models/account_analytic.py

```diff
@@ -27,7 +27,10 @@ def create(self, vals_list):
                 raise UserError(_('You cannot create timesheets for a task that is linked to a time off type. Please use the Time Off application to request new time off instead.'))
         return super().create(vals_list)
 
+    def _get_whitelist_fields_to_write(self):
+        return []
+
     def write(self, vals):
-        if not self.env.su and self.filtered('holiday_id'):
+        if not self.env.su and self.filtered('holiday_id') and any(key not in self._get_whitelist_fields_to_write() for key in list(vals.keys())):
```

Comment:
Okay, we will do something else then. 🙂 

It would be better to create a method to check if the user can write on a "time off timesheet" (timesheet generated by a time off). By doing that, we will reduce the chance we remove that method in an eventual refactoring (I think 🤞🏼). 🙂 

And so we could do something like this:

```suggestion
    def _check_can_update_timesheet(self):
        return self.env.su or not self.filtered('holiday_id')

    def write(self, vals):
        if not self._check_can_update_timesheet():
```

Normally by doing that you can fix your issue in your module, right? 🤞🏼 🙂